examples/multi_deployment/main.tf

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  credentials_file_path = var.credentials_path
+}
+/*****************************************
+  Folder Structure
+ *****************************************/
+module "migration-folders" {
+  source  = "terraform-google-modules/folders/google"
+  version = "~> 2.0"
+  # parent = "organizations/${var.organization_id}"
+  parent            = "folders/619261348000"
+  names             = ["core-project", "velo", "prod", "stg", "test"]
```

Comment:
Let's make the folders driven by an "environments" variable, with default of:

- nonprod
- prod
- shared (including velostrata)

Reply:
Done; I made an env var to pass in all environments except shared since shared includes vpc and velostrata projects that is always necessary